README.md

```diff
@@ -248,91 +280,137 @@ const senderWallet = generationResult.wallet;
+Utils.isValidAddress(rippleClassicAddress) // returns true
+Utils.isValidAddress(rippleXAddress) // returns true
+Utils.isValidAddress(bitcoinAddress) // returns false
 ```
 
 You can also validate if an address is an X-Address or a classic address.
+
 ```javascript
-const { Utils } = require("xpring-js")
+const { Utils } = require('xpring-js')
 
-const rippleClassicAddress = "rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G"
-const rippleXAddress = "X7jjQ4d6bz1qmjwxYUsw6gtxSyjYv5iWPqPEjGqqhn9Woti";
-const bitcoinAddress = "1DiqLtKZZviDxccRpowkhVowsbLSNQWBE8";
+const rippleClassicAddress = 'rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G'
+const rippleXAddress = 'X7jjQ4d6bz1qmjwxYUsw6gtxSyjYv5iWPqPEjGqqhn9Woti'
+const bitcoinAddress = '1DiqLtKZZviDxccRpowkhVowsbLSNQWBE8'
 
-Utils.isValidXAddress(rippleClassicAddress); // returns false
-Utils.isValidXAddress(rippleXAddress); // returns true
-Utils.isValidXAddress(bitcoinAddress); // returns false
+Utils.isValidXAddress(rippleClassicAddress) // returns false
+Utils.isValidXAddress(rippleXAddress) // returns true
+Utils.isValidXAddress(bitcoinAddress) // returns false
 
-Utils.isValidClassicAddress(rippleClassicAddress); // returns true
-Utils.isValidClassicAddress(rippleXAddress); // returns false
-Utils.isValidClassicAddress(bitcoinAddress); // returns false
+Utils.isValidClassicAddress(rippleClassicAddress) // returns true
+Utils.isValidClassicAddress(rippleXAddress) // returns false
+Utils.isValidClassicAddress(bitcoinAddress) // returns false
 ```
 
 ### X-Address Encoding
 
 You can encode and decode X-Addresses with the SDK.
 
 ```javascript
-const { Utils } = require("xpring-js")
+const { Utils } = require('xpring-js')
 
-const rippleClassicAddress = "rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G"
-const tag = 12345;
+const rippleClassicAddress = 'rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G'
+const tag = 12345
 
 // Encode an X-Address.
-const xAddress = Utils.encodeXAddress(rippleClassicAddress, tag); // X7jjQ4d6bz1qmjwxYUsw6gtxSyjYv5xRB7JM3ht8XC4P45P
+const xAddress = Utils.encodeXAddress(rippleClassicAddress, tag) // X7jjQ4d6bz1qmjwxYUsw6gtxSyjYv5xRB7JM3ht8XC4P45P
 
 // Decode an X-Address.
-const decodedClassicAddress = Utils.decodeXAddress(xAddress);
+const decodedClassicAddress = Utils.decodeXAddress(xAddress)
+
+console.log(decodedClassicAddress.address) // rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G
+console.log(decodedClassicAddress.tag) // 12345
+```
+
+## Usage: PayID
+
+Two classes are used to work with PayID: `PayIdClient` and `XrpPayIdClient`.
 
-console.log(decodedClassicAddress.address); // rnysDDrRXxz9z66DmCmfWpq4Z5s4TyUP3G
-console.log(decodedClassicAddress.tag); // 12345
+### PayIdClient
+
+`PayIdClient` can resolve addresses on arbitrary cryptocurrency networks.
+
+```javascript
+import { PayIdClient } from 'xpring-js'
+
+// Resolve on Bitcoin Mainnet.
+const network = 'btc-mainnet'
+const payIdClient = new PayIdClient(network)
+
+const payId = 'georgewashington$xpring.money'
```

Comment:
Does this PayID exist?

Reply:
It doesn't. I wouldn't expect these examples to necessarily work (i'd expect our demos to though).